nosql/JobListing.json

```diff
@@ -0,0 +1,17 @@
+{
+  "jobID":"Integer",
+  "industryType":"String",
+  "employmentType":"String",
+  "jobFunctions":"String",
+  "positionName":"String",
+  "professionLevel":"String",
+  "companyID":"Number",
+  "companyName":"String",
+  "companyLocation":"String",
+  "compnayPicture":"URL/String",
+  "relatedConnections":[{},{}],
+  "jobBrief":"String",
+  "requiredSkills":["String","String"],
```

Comment:
["String"] can be enough if we are just listing the names but maybe if names weren't unique an id referencing skills in SQL will be more handy --> ["Number"] , What do you think?